.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,77 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.194.0/containers/python-3
+{
+	"name": "Python 3",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"context": "..",
+		"args": {
+			// Update 'VARIANT' to pick a Python version: 3, 3.6, 3.7, 3.8, 3.9
+			"VARIANT": "3.8",
+			// Options
+			"NODE_VERSION": "none"
+		}
+	},
+	"runArgs": [
+        // Enable GPU passthrough, requires WSL2 on Windows
+		//"--gpus=all",
+	],
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"editor.rulers": [
+			120
+		],
+		"python.pythonPath": "/usr/local/bin/python",
+		"python.defaultInterpreterPath": "/usr/local/bin/python",
+		"python.languageServer": "Pylance",
+		"python.autoComplete.addBrackets": true,
+		"python.analysis.autoImportCompletions": true,
+		"python.analysis.completeFunctionParens": true,
+		"python.analysis.autoSearchPaths": true,
+		"python.analysis.useImportHeuristic": true,
+		"python.linting.enabled": true,
+		"python.linting.pylintEnabled": false,
+		"python.linting.flake8Enabled": true,
+		"python.formatting.autopep8Path": "/usr/local/py-utils/bin/autopep8",
+		"python.formatting.blackPath": "/usr/local/py-utils/bin/black",
+		"python.formatting.yapfPath": "/usr/local/py-utils/bin/yapf",
+		"python.formatting.provider": "black",
+		"python.formatting.blackArgs": [
+			"--line-length=120"
+		],
+		"python.linting.banditPath": "/usr/local/py-utils/bin/bandit",
+		"python.linting.flake8Path": "/usr/local/py-utils/bin/flake8",
+		"python.linting.mypyPath": "/usr/local/py-utils/bin/mypy",
+		"python.linting.pycodestylePath": "/usr/local/py-utils/bin/pycodestyle",
+		"python.linting.pydocstylePath": "/usr/local/py-utils/bin/pydocstyle",
+		"python.linting.pylintPath": "/usr/local/py-utils/bin/pylint",
+		"python.linting.flake8Args": [
+			"--ignore=E24,W504,F401,E203,W503",
+			"--max-line-length=120",
+			"--verbose"
+		],
+		"restructuredtext.confPath": "${workspaceFolder}/docs/source",
+		"restructuredtext.builtDocumentationPath": "${workspaceFolder}/docs/build",
+		"restructuredtext.languageServer.disabled": false
+	},
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [
+		"ms-python.python",
+		"ms-python.vscode-pylance",
+		"visualstudioexptteam.vscodeintellicode",
+		"kevinrose.vsc-python-indent",
+		"littlefoxteam.vscode-python-test-adapter",
+		"hbenl.vscode-test-explorer",
+		"medo64.render-crlf",
+		"shardulm94.trailing-spaces",
+		"njqdev.vscode-python-typehint",
+		"lextudio.restructuredtext",
+		"trond-snekvik.simple-rst",
```

Comment:
why do we need 2 different rst plugins?

Reply:
The first provides live preview, linting and intellisense, the second one syntax highlighting